### ISSUE TITLE
[v1][Feature Request] Mini QDrawer open as overlay

### ISSUE THREAD

**Issue by smakinson** [quasarframework/quasar#3945]
I was experimenting with an idea based on the example in the docs where the left drawer goes from mini to not mini on hover. I was able to have it change to overlay also on hover, but when I use this method to accomplish it, that min mode width ( default 60px ) is lost when it switches to overlay mode. I'd like to be able to keep that spacing to prevent movement of the main content while also having it "open" ( from mini to not mini ) as overlay, also to prevent movement of the main content. I may be going about this in the wrong way ( also I'm still on 0.17 ) so I thought I'd start a discussion about it.

Codepen showing what happens now:
https://codepen.io/smakinson/pen/VNBxXy?editors=1010

A side note to this ( and maybe another discussion ) is I have also been experimenting with multiple drawers on the right side and trying to have one overlay the other that would be a mini to not mini expanded. One would be a navigation type drawer and the other potentially a chat area. Anyway, not to talk too much on that one other than to see if I should open a discussion or if the idea of multiple drawers on one side is an unexpected move.

**Comment by devel-pa** [comment]
Hi,
My fix for this on `style` area:
```
.q-drawer-container aside {
  transform: translate3d(0px, 0px, 0px) !important;
}
```

**Comment by smakinson** [comment]
@devel-pa I added the above css to the pen & I am seeing the same issue.

**Comment by devel-pa** [comment]
@smakinson Sorry, an incomplete `style`, my bad. Tested that adding into your pen. The commented second one is for right drawer.

```
.q-page-container {
  padding-left: 57px;
  /*padding-right: 57px;*/
}
```

**Comment by rstoenescu** [comment]
Fixed through #4500
